pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -193,6 +193,21 @@ func (g *GadgetTracerManager) DumpState(_ context.Context, req *pb.DumpStateRequ
 	return &pb.Dump{State: out}, nil
 }
 
+func addFanotifyOptions(opts []containercollection.ContainerCollectionOption, conf *Conf) []containercollection.ContainerCollectionOption {
+	opts = append(opts, containercollection.WithRuncFanotify())
+	opts = append(opts, containercollection.WithInitialKubernetesContainers(conf.NodeName))
+	if conf.FallbackPodInformer {
+		log.Infof("GadgetTracerManager: enabling fallback podinformer")
+		opts = append(opts, containercollection.WithFallbackPodInformer(conf.NodeName))
+	}
```

Comment:
If we move this here, we will not add the fallback when using NRI or CRI-O (`conf.HookMode` is "none"). Or what did I miss?

Reply:
You are right, I missed that `none` was a valid option when doing the refactor. I guess I'll have to do this differently, then.